test/integration/targets/sefcontext/aliases

```diff
@@ -0,0 +1,3 @@
+needs\/privileged
+needs\/root
```

Comment:
Also, have you verified that `needs/privileged` is necessary? You'd have to run the tests in docker on a host with selinux support to see if docker privileged mode is necessary. Otherwise, leave out `needs/privileged` until we know it's needed.